taskcluster/ffios_taskgraph/transforms/bitrise.py

```diff
@@ -31,24 +29,6 @@ def set_bitrise_app(config, tasks):
         yield task
 
 
-@transforms.add
-def set_build_params(config, tasks):
-    build_params = {}
-
-    if config.params["commit_message"]:
-        build_params["commit_message"] = config.params["commit_message"]
-
-    if config.params["pull_request_number"]:
-        build_params["pull_request_id"] = config.params["pull_request_number"]
-
-    if not build_params:
-        yield from tasks
-    else:
-        for task in tasks:
-            task["bitrise"].setdefault("build_params", {}).update(build_params)
-            yield task
-
-
```

Comment:
Oh crap, I overlooked these in my review :(

These are here because `commit_message` and `pull_request_number` aren't official Taskgraph parameters (firefox-ios extends the parameters schema). So now we have no way of forwarding them to bitrise.

Since the `bitriscript` change is already live, let's deploy this now. But I think we'll need to re-add the ability to specify other global build_params afterwards